woocommerce/woocommerce-android
# [Customer Search] Open a list in the top position when first page is loaded

Closes: #9379


### Description

Resets scrolling position every time the first page is loaded, including when activity was killed

### Testing instructions

* Start creating and order
* Add a customer
* Scroll down in the list
* Hide activity and open with "don't keep activities"
* Notice that the scrolling state is restarted


Also,

* Scroll down on one search query
* Change query or search type
* Returned back 
* Notice that the scrolling state is restarted

### Images/gif



https://github.com/woocommerce/woocommerce-android/assets/4923871/11ada588-de2e-463b-a50f-fa78ddbf0b5b
